### PR TITLE
checkquery refactory

### PR DESCRIPTION
**Pull request by daveads**
refactored
* checkquery

### CODE REVIEW

**Comment by andrewtavis** [comment]
@daveads, do you have any idea what's causing the error here?

```
============================= test session starts ==============================
platform linux -- Python 3.9.19, pytest-8.1.1, pluggy-1.5.0
rootdir: /home/runner/work/Scribe-Data/Scribe-Data
configfile: pyproject.toml
plugins: cov-5.0.0
collected 63 items / 1 error

==================================== ERRORS ====================================
________________ ERROR collecting tests/load/test_checkquery.py ________________
ImportError while importing test module '/home/runner/work/Scribe-Data/Scribe-Data/tests/load/test_checkquery.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.9.[19](https://github.com/scribe-org/Scribe-Data/actions/runs/8843271753/job/24286017797?pr=131#step:8:20)/x64/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/load/test_checkquery.py:10: in <module>
    from scribe_data.checkquery import (
src/scribe_data/checkquery.py:36: in <module>
    from checkquery_utills.constants import *
E   ModuleNotFoundError: No module named 'checkquery_utills'
=========================== short test summary info ============================
ERROR tests/load/test_checkquery.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
```

**Comment by andrewtavis** [comment]
Basic comments on this is that the directory name should be `checkquery_utils`, not `utills`. I'm also assuming that we're getting the error that the files aren't able to be found because there's no `__init__.py` in the new directory :)

File: src/scribe_data/checkquery_utills/checkquer.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
+import argparse
```
Comment (andrewtavis) [inline src/scribe_data/checkquery_utills/checkquer.py:1]: File should be named `checkquery`

Comment (andrewtavis) [reply src/scribe_data/checkquery_utills/checkquer.py:1]: I actually think a better name would even be `check`.

**Comment by andrewtavis** [comment]
Let me know on the above, @daveads! :)

**Comment by daveads** [comment]
> Let me know on the above, @daveads! :) I'll pick up more of the review after that 😊

i will fix this before next week @andrewtavis

**Comment by andrewtavis** [comment]
Thank you, @daveads! Looking forward to it :)

**Review by andrewtavis — APPROVED** [review]
Thanks for bringing the structural problems to our attention and working on them, @daveads! Appreciate it 😊

**Pull request merged**